addons/web/static/tests/views/fields/parsers.test.js

```diff
@@ -0,0 +1,195 @@
+import { before, expect, test } from "@odoo/hoot";
+import { makeMockEnv, patchWithCleanup } from "../../web_test_helpers";
+
+import {
+    parseFloat,
+    parseFloatTime,
+    parseInteger,
+    parseMonetary,
+    parsePercentage,
+} from "@web/views/fields/parsers";
+import { localization } from "@web/core/l10n/localization";
+import { nbsp } from "@web/core/utils/strings";
+import { session } from "@web/session";
+
+before(async () => {
+    await makeMockEnv();
+});
+
+test("parseFloat", () => {
```

Comment:
what would you think about having a sub suite for each function, and split those kind of tests into multiple tests?